### PR TITLE
N1QL -> SQL++ rebrand

### PR DESCRIPTION
**Pull request by tom-rosewell**
[DOC-10414](https://issues.couchbase.com/browse/DOC-10414)

### CODE REVIEW

File: modules/concept-docs/pages/analytics-for-sdk-users.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 = Analytics
-:description: Parallel data management for complex queries over many records, using a familiar N1QL-like syntax.
+:description: Parallel data management for complex queries over many records, using a syntax similar to {sqlpp} (formerly N1QL).
```
Comment (RichardSmedley) [inline modules/concept-docs/pages/analytics-for-sdk-users.adoc]: I think this needs a reword, now that they are both SQL++

@simon-dew What do you think?

Maybe we can say "using a familiar {sqlpp} syntax"
It's not, in any case, a good place for the "formerly N1QL" snippet.

File: modules/concept-docs/pages/data-model.adoc [hunk 2]
```diff
@@ -23,7 +23,7 @@ Values can be placed in the beginning or end of a list, and can be accessed usin
 * *Queue* is a wrapper over a _list_ which offers FIFO (first-in-first-out) semantics, allowing it to be used as a lightweight job queue.
 * *Set* is a wrapper over a _list_ which provides the ability to handle unique values.
 
-These data structures are stored as JSON documents in Couchbase, and can therefore be accessed using N1QL, Full Text Search, and normal key-value operations.
+These data structures are stored as JSON documents in Couchbase, and can therefore be accessed using {sqlpp} (formerly N1QL), Full Text Search, and normal key-value operations.
```
Comment (RichardSmedley) [inline modules/concept-docs/pages/data-model.adoc]: Fine to leave in - but note just replacing N1QL with Query may also be as effective in cases like these.

File: modules/concept-docs/pages/analytics-for-sdk-users.adoc (hunk 1 above)
Comment (simon-dew) [reply modules/concept-docs/pages/analytics-for-sdk-users.adoc]: I agree in this case -- `using a familiar {sqlpp} syntax`

File: modules/howtos/pages/analytics-using-sdk.adoc [hunk 3]
```diff
@@ -1,5 +1,5 @@
 = Analytics using the SDK
-:description: Parallel data management for complex queries over many records, using a familiar N1QL-like syntax.
+:description: Parallel data management for complex queries over many records, using a familiar {sqlpp}-like syntax.
```
Comment (RichardSmedley) [inline modules/howtos/pages/analytics-using-sdk.adoc]: See also the comment above, but probably best simply as 
"using a familiar {sqlpp} syntax."

File: modules/howtos/pages/n1ql-queries-with-sdk.adoc [hunk 4]
```diff
@@ -237,13 +237,13 @@ include::howtos:example$n1ql_ops.py[tag=scope]
...
-* The http://query.pub.couchbase.com/tutorial/#1[N1QL interactive tutorial] is a good introduction to the basics of N1QL use.
+* For a deeper dive into {sqlpp} from the SDK, refer to our xref:concept-docs:n1ql-query.adoc[{sqlpp} SDK concept doc].
+* The xref:{version-server}@server:n1ql:n1ql-language-reference/index.adoc[Server doc {sqlpp} intro] introduces a complete guide to the {sqlpp} language, including all of the latest additions.
+* The http://query.pub.couchbase.com/tutorial/#1[{sqlpp} interactive tutorial] is a good introduction to the basics of {sqlpp} use.
 * For scaling up queries, be sure to
 xref:{version-server}@server:n1ql:n1ql-language-reference/index.adoc[read up on Indexes].
-* N1QL is for operational queries; for analytical workloads, read more on xref:concept-docs:http-services.adoc#Long-Running-Queries-&-Big-Data[when to choose Analytics Service], our implementation of SQL++.
+* {sqlpp} is for operational queries; for analytical workloads, read more on xref:concept-docs:http-services.adoc#Long-Running-Queries-&-Big-Data[when to choose Analytics Service].
```
Comment (RichardSmedley) [inline modules/howtos/pages/n1ql-queries-with-sdk.adoc]: Maybe change to "The Query Service"

File: modules/project-docs/pages/3.0-pre-release-notes.adoc [hunk 5]
```diff
@@ -32,7 +32,7 @@ pip install couchbase==3.0.0b3
 Bumped LCB requirement to 3.0.0.
 
 * https://issues.couchbase.com/browse/PYCBC-757[PYCBC-757]:
-Refactored for FTS/N1QL/durability LCB changes.
+Refactored for FTS/{sqlpp} (N1QL)/durability LCB changes.
```
Comment (RichardSmedley) [inline modules/project-docs/pages/3.0-pre-release-notes.adoc]: Please revert these - release notes are an historic record.

File: modules/project-docs/pages/3.0-pre-release-notes.adoc [hunk 6]
```diff
@@ -58,7 +58,7 @@ Implemented replica reads according to RFC.
 Refactored durability params to prevent impossible combinations.
 
 * https://issues.couchbase.com/browse/PYCBC-639[PYCBC-639]:
-Example N1QL statements do seem to work in "Hello Couchbase" example.
+Example {sqlpp} statements do seem to work in "Hello Couchbase" example.
```
Comment (RichardSmedley) [inline modules/project-docs/pages/3.0-pre-release-notes.adoc]: Please revert these - release notes are an historic record.

File: modules/project-docs/pages/3.0-pre-release-notes.adoc [hunk 7]
```diff
@@ -153,7 +153,7 @@ Update sphinx-autodoc-typehints to attempt to use type comments
 Get rid of obsolete coll/scope injection calls
 
 * https://issues.couchbase.com/browse/PYCBC-683[PYCBC-683]:
-N1QL Query Index Management
+{sqlpp} Query Index Management
```
Comment (RichardSmedley) [inline modules/project-docs/pages/3.0-pre-release-notes.adoc]: Please revert these - release notes are an historic record.

**Comment by RichardSmedley** [comment]
Looks good - just a couple of changes needed,
and some input on the Analytics comparison.

**Review by RichardSmedley — APPROVED** [review]
LGTM :+1:

**Pull request merged**